src/Search.js

```diff
@@ -13,11 +14,10 @@ const Search = () => {
             <div className="search-row">
               <input
                 type="text"
-                id="customerName"
                 className="form-control"
-                placeholder="Customer Id"
+                placeholder="Coustomer ID"
               />
-              <button className="btn btn-primary">Search IDs</button>
+              <Button content="search IDs" />
```

Comment:
Here we're passing a `content` prop but it's not being used in the component at all. (You can test this by changing it to be `content="This doesn't show!"`).

How could we fix that?